pages/chat/_entity.vue

```diff
@@ -23,25 +23,26 @@
 
           <bottom-sheet :items="bottomSheetItems" ref="mobileDialog"/>
         </template>
-
-        </div>
+      </div>
     </header-bar>
 
     <!-- Main Section -->
     <main class="d-flex flex-row flex-grow-1 hide-overflow">
-      <div class="d-flex flex-column flex-grow-1 os-host-flexbox">
+      <div class="d-flex flex-column flex-grow-1">
         <!-- Message List -->
-        <overlay-scrollbars
+        <simplebar
+          class="simplebar wide-scrollbar flex-grow-1" 
+          data-simplebar-auto-hide="false" 
+          data-simplebar-force-visible="true"
           ref="messageList"
-          class="flex-grow-1 flex-shrink-1 wide-scrollbar"
           :options="{scrollbars:{clickScrolling: true}}">
```

Comment:
The `options` attribute does nothing to `<simplebar/>`  components.
```suggestion
          >
```
By the way, does click scrolling work on simplebar (and how)?

Reply:
>By the way, does click scrolling work on simplebar (and how)?

Yes, it's somewhat weird though. It doesn't snap to the destination but moves towards it in steps. There's no setting for this AFAIK so we'll probably have to make an issue/PR.